Add  `--quiet` flag to suppress most log messages (#846)
SwiftGen/SwiftGen: Documentation/ConfigFile.md

  swiftgen config init --config tools/swiftgen/swiftgen-config.yml
  ```
  
* Change the default log level to quiet or verbose. If no log level is specified, default will be used.

>   * Verbose mode, which will print every command being executed when executing it, using the `--verbose` flag. This allows your to:

Reviewer: ```suggestion
  * Verbose mode (`--verbose` or `-v` flag), which will print every command being executed when executing it. This allows your to:
```

Author: Done

    * control what is being run, by logging what happens during execution
    * know the equivalent command to type if you were to run each swiftgen command manually instead of using the config file — which can be useful if you need to debug or tweak a particular command in isolation for example

      ```sh
      swiftgen config run --verbose
      ```

>   * Quiet mode, which will silent every command being executed when executing it (except for errors):

Reviewer: ```suggestion
  * Quiet mode (`--quiet` or `-q` flag), which will silent every command being executed when executing it (except for errors):
```

Author: Done

>   * Quiet mode (`--quiet` or `-q` flag), which will silent every command being executed when executing it (except for errors):

Reviewer: ```suggestion
  * Quiet mode (`--quiet` or `-q` flag), which will silence all logs (except for errors):
```

  swiftgen config init --config tools/swiftgen/swiftgen-config.yml
  ```
  

> * Enable the verbose mode, which will print every command being executed when executing it, using the `--verbose` flag. This allows your to:

Reviewer: Obviously tweak docs regarding the `--quiet` flag

Author: Made the changes
